src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -248,7 +244,8 @@ export default {
 	mounted() {
 		this.getRecommendedLoans();
 		this.getSecondCategoryData();
-		this.fetchMFILoans();
+		this.verifySpotlightIndex();
+		this.fetchSpotlightLoans();
```

Comment:
Long-term we'll probably want to only run this fetch when needed (when the spotlight is visible), but we have another ticket for that lazy loading, and this query isn't adding a new query compared to before.